### ISSUE TITLE
Client can only connect to Server and Download the files,is there a way client can upload a file to server?

### ISSUE THREAD

**Issue by noob09** [sebsto/tsunami-udp#9]
Currently Client can only connect to Server and Download the files,
is there a way client can upload a file to server?(other way around).
great work guys ! appreciate your help

**Comment by ohbe1jacobi** [comment]
No, it's one way.  Nothing stopping you from running a server on both ends though
